media/js/fonts.js

```diff
@@ -0,0 +1,158 @@
+(function(win, doc, $) {
+    /*
+        Font Loading
+    */
+
+    if(window.waffle && window.waffle.flag_is_active('web_font_loader')) {
+
+        // what fonts are we loading?
+        var fonts = {
```

Comment:
I think you want this to be an array and not an object.  So....

```
var fonts = [{  }, {  }]
```

Objects are good for key/value